extensions/slack/src/search.tsx

```diff
@@ -19,12 +30,33 @@ function Search() {
         const isUser = item.id.startsWith("U");
 
         if (isUser) {
-          const { id: userId, name, icon, teamId: workspaceId, conversationId } = item as User;
+          const {
+            id: userId,
+            name,
+            icon,
+            title,
+            statusEmoji,
+            statusText,
+            statusExpiration,
+            teamId: workspaceId,
+            conversationId,
+            timezone,
+          } = item as User;
           return (
             <List.Item
               key={userId}
               title={name}
+              subtitle={title}
               icon={icon ? { source: icon, mask: Image.Mask.Circle } : Icon.Person}
+              accessories={[
+                {
+                  icon: convertSlackEmojiToUnicode(statusEmoji ?? ""),
+                  text: statusText ?? "",
+                  tooltip: statusExpiration ?? 0,
+                },
+                { icon: Icon.Globe, text: timezone.split("/")[1].replace(/_/g, " ") },
+                { icon: Icon.Clock, text: getCoworkerTime(timezone) },
+              ]}
```

Comment:
Suggestion: As you already indicated, a preference to show the metadata might keep things backwards-compatible. Beyond that future-proofing with granular options for specific metadata like AM/PM vs 24-hr time is a good-to-have but not really necessary imo (but I could be wrong about this). If folks create an issue requiring specific metadata prefs like that, we can handle it with iterative PRs.

Also, accessory has a date option which takes `Date` object and shows relative info. Should we use that here with more info in the `tooltip` of that accessory?

Reply:
I attempted `Date` accessory but found that it wasn't super useful to show the time as "3hr" as it didn't portray what the time actually means related to your current time. If you can think of another way to make that work I'll take another stab.

Reply:
Yeah I thought about it and it is not applicable to what we are trying to achieve here, you can resolve this!